lib/src/main/java/com/diffplug/spotless/JarState.java

```diff
@@ -42,8 +42,7 @@
 public final class JarState implements Serializable {
 	private static final long serialVersionUID = 1L;
 
-	@SuppressWarnings("unused")
-	private final Set<String> mavenCoordinates;
+	private final TreeSet<String> mavenCoordinates;
```

Comment:
Sorry, I almost missed this one.  There are corner-cases where classpath order matters.  Unless we have a really compelling reason, I think we should keep this as `Set` and not `TreeSet`

Reply:
Just wanted to emphasize that I expect a deterministic result for getMavenCoordinates for the same input. I do not insist. What's your final decision?

Reply:
Set

Reply:
then feel free to press the merge button, and I'll test the `master` integration build on some of my projects

Reply:
Ok, THX.